heapoverflow/heapoverflow.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "HeapOverflow",
```

In the docs: `key` should be a unique lowercase alphanumeric string with no special characters:
```suggestion
    "key": "heapoverflow",
```

The CHEESEHub UI probably needs additional steps to better validate this one (sorry!)